index.js

```diff
@@ -1600,30 +1604,81 @@ Auth0.prototype.requestSMSCode = function (options, callback) {
     throw new Error('A callback function is required');
   }
 
-  assert_required(options, 'apiToken');
   assert_required(options, 'phone');
+  options.phoneNumber = options.phone;
+  delete options.phone;
 
-  var apiToken = options.apiToken;
-  var phone = options.phone;
+  return this.startPasswordless(options, callback);
+};
+
+/**
+ * Send email or SMS to do passwordless authentication
+ *
+ * @example
+ *     // To send an email
+ *     auth0.startPasswordless({email: 'foo@bar.com'}, function (err, result) {
+ *       if (err) return console.log(err.error_description);
+ *       console.log(result);
+ *     });
+ *
+ * @example
+ *     // To send a SMS
+ *     auth0.startPasswordless({phoneNumber: '+14251112222'}, function (err, result) {
+ *       if (err) return console.log(err.error_description);
+ *       console.log(result);
+ *     });
+ *
+ * @method startPasswordless
+ * @param {Object} options
+ * @param {Function} callback
+ */
+
+Auth0.prototype.startPasswordless = function (options, callback) {
+  if ('object' !== typeof options) {
+    throw new Error('An options object is required');
+  }
+  if ('function' !== typeof callback) {
+    throw new Error('A callback function is required');
+  }
+  if (!options.email && !options.phoneNumber) {
+    throw new Error('An email or a phone number is required.');
```

Comment:
I'd improve this later as "An `email` or `phoneNumber` is required"... so they know from the error that `phoneNumber` is the name of the var. Not a big deal now.